src/NBitcoin.Tests/util_tests.cs

```diff
@@ -7,24 +7,39 @@
 using NBitcoin.BouncyCastle.Math;
 using NBitcoin.Crypto;
 using NBitcoin.DataEncoders;
+using NBitcoin.Networks;
 using NBitcoin.OpenAsset;
 using Newtonsoft.Json.Linq;
+using Stratis.Bitcoin.Tests.Common;
 using Xunit;
 
 namespace NBitcoin.Tests
 {
-    public class util_tests
+    public class Util_Tests
     {
+        private readonly Network networkMain;
+        private readonly Network networkTestNet;
+        private readonly Network networkRegTest;
+
         private static byte[] ParseHex_expected = new byte[]{
     0x04, 0x67, 0x8a, 0xfd, 0xb0, 0xfe, 0x55, 0x48, 0x27, 0x19, 0x67, 0xf1, 0xa6, 0x71, 0x30, 0xb7,
     0x10, 0x5c, 0xd6, 0xa8, 0x28, 0xe0, 0x39, 0x09, 0xa6, 0x79, 0x62, 0xe0, 0xea, 0x1f, 0x61, 0xde,
     0xb6, 0x49, 0xf6, 0xbc, 0x3f, 0x4c, 0xef, 0x38, 0xc4, 0xf3, 0x55, 0x04, 0xe5, 0x1e, 0xc1, 0x12,
     0xde, 0x5c, 0x38, 0x4d, 0xf7, 0xba, 0x0b, 0x8d, 0x57, 0x8a, 0x4c, 0x70, 0x2b, 0x6b, 0xf1, 0x1d,
     0x5f};
 
+        public Util_Tests()
+        {
+            NetworkRegistration.Clear();
```

Comment:
Why the need to clear NetworkRegistration?

Reply:
@dangershony I thought it a useful method should we wish to do however, the reason is a bit complex...

Prior to this change NetworkRegistration.GetNetworks yield returned the networks in this order:

```
public static IEnumerable<Network> GetNetworks()
{
    yield return NBitcoin.Networks.Main;
    yield return NBitcoin.Networks.TestNet;
    yield return NBitcoin.Networks.RegTest;
    ...
```

Because we can now potentially have Stratis network's in the collection AS WELL as the fact that the main networks (StratisMain and BitcoinMain) share the same `Bech32Encoder` set:

```
    var encoder = new Bech32Encoder("bc");
    this.Bech32Encoders = new Bech32Encoder[2];
    this.Bech32Encoders[(int)Bech32Type.WITNESS_PUBKEY_ADDRESS] = encoder;
    this.Bech32Encoders[(int)Bech32Type.WITNESS_SCRIPT_ADDRESS] = encoder;
```
 when we try and get the network type from the base address it sometime returns BitcoinMain and sometimes StratisMain. So I needed to clear the container before the test start.